StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/PaymentMethodMessagingElement/PMME+API.swift

```diff
@@ -39,6 +39,16 @@ extension PaymentMethodMessagingElement {
                 case .success(let apiResponse):
                     continuation.resume(returning: apiResponse)
                 case .failure(let error):
+                    if let decodingError = error as? DecodingError,
+                        case let .keyNotFound(codingKey, context) = decodingError {
+                        stpAssertionFailure(context.debugDescription)
+                        let problemKey = codingKey.stringValue
+                        let problemPath = context.codingPath
+                        let fullPath = problemPath.reduce("") { $0 + $1.stringValue + "." } + problemKey
+                        let error = PaymentMethodMessagingElementError.unexpectedResponseFromStripeAPI
```

Comment:
Is this necessary? If you just send the decodingError as-is what happens? I think/hope it sends useful enough info by default.